### DISCUSSION TITLE
Import Completion status Api

### DISCUSSION THREAD

**Discussion by MALKARAJ** [osm-search/Nominatim#3145] [Q&A]
is there any way to check whether import process is completed with success note or not

**Comment by mtmail** [comment]
There's `nominatim admin --check-database` which looks if all places are indexed and if the required search indices have be created. It's not an API, usually the webserver configuration is separate from the importing of data.

**Reply by mtmail** [reply]
The `.../status?format=json` URL reports if the webserver is running, if it can connect to the database and if the database has any places imported. It doesn't check how many places though.

**Reply by MALKARAJ** [reply]
you mean nominatim admin --check-database which tell if some places are not indexed

**Reply by mtmail** [reply]
Yes. It's a useful tool, but it's not an API.
